lib/build.js

```diff
@@ -124,7 +124,8 @@ module.exports = {
       args: [],
       cwd: path,
       sh: true,
-      errorMatch: ''
+      errorMatch: '',
+      errorMatchPathAbsolute: false
```

Comment:
To avoid having more configuration options (which tend to bloat the README and files) I want to suggest having a different name for the matched group in the regex instead. It could be called `absFile`. When error matching, `atom-build` should check if `file` is matched, in that case open `cwd + file`, otherwise of `absFile` is set, open `absFile`.